tests/unit/IO/test_vt_writer.py

```diff
@@ -140,7 +140,7 @@ def test_vt_writer_required_fields_output(self):
             output_file_name = f"{output_file_prefix}.{i}.json"
             output_file = os.path.join(output_dir, output_file_name)
 
-            print(f"[{__loader__.name}] Compare input file ({input_file_name}) and output file ({output_file_name})...")
+            # print(f"[{__loader__.name}] Compare input file ({input_file_name}) and output file ({output_file_name})...")
```

Comment:
Why comment this print statement, rather than eliminate it if no longer needed?

Reply:
Fixed--I removed the statement